doc/api/util.md

```diff
@@ -1550,25 +1547,22 @@ const options = {
   'logfile': { type: 'string' },
   'no-logfile': { type: 'boolean' },
 };
-const { values, tokens } = parseArgs({ options, tokens: true });
+const { values, tokens = [] } = parseArgs({ options, tokens: true });
```

Comment:
This does not seem needed for this example. Always get an array back if ask for `tokens`?
```suggestion
const { values, tokens } = parseArgs({ options, tokens: true });
```